mvp-0/README.md

```diff
@@ -1,117 +1,239 @@
 # MVP-0
 
-## Deployment
-
-### Observer cluster
-
-#### Deploy dnation-kubernetes-monitoring-stack without dnation-kubernetes-monitoring
-
-1. Optional: Fill `thanos-storage.yaml` with bucket credentials in **thanosStorage config**
-2. Deploy
-   ```bash
-   helm repo add dnationcloud https://dnationcloud.github.io/helm-hub/
-   helm repo update dnationcloud
-   helm upgrade --install dnation-kubernetes-monitoring-stack dnationcloud/dnation-kubernetes-monitoring-stack \
-     --set dnation-kubernetes-monitoring.enabled=false \
-     -f values-observer.yaml \
-     -f thanos-storage.yaml # Optional
-   ```
-
-#### Deploy dnation-kubernetes-monitoring
-
-1. Clone and build rules/dashboards
-   ```bash
-   git clone https://github.com/dNationCloud/kubernetes-monitoring.git
-   cd kubernetes-monitoring
-   git checkout scs-kaas-mvp
-   make jsonnet-package
-   cd -
-   ```
-2. Deploy
-   ```bash
-   helm upgrade --install dnation-kubernetes-monitoring kubernetes-monitoring/chart --dependency-update \
-     --set releaseOverride=dnation-kubernetes-monitoring-stack \
-     -f values-observer-dash.yaml
-   ```
-
-#### Deploy kaas-metric-importer
-
-Deployment uses an image built from https://github.com/m3dbx/prometheus_remote_client_golang.
+This MVP-O aims to build an Observer monitoring solution intended to offer a global **metrics**
+view of the CSP infrastructure. It is the platform where CSP infrastructure **metrics**
+are fetched, processed, stored, and visualized. Note that this monitoring solution could
+be extended, and the other two observability signals (logs and traces) from the CSP
+infrastructure could also be processed here.
+
+_This MVP-0 builds a base for an Observer monitoring solution intended to become an **SCS product**
+in the future versions, once it attains sufficient stability._
+
+The current MVP-0 implements the initial iteration of the Observer monitoring solution,
+deployed in the Kubernetes cluster, subsequently referred to as the Observer cluster.
+As illustrative examples of how this monitoring solution can be utilized, the MVP-0 also
+implements three use cases:
+- Monitoring of the KaaS layer
+- Monitoring of the IaaS layer
+- Monitoring of infrastructure services (such as CSP services deployed on top of the IaaS layer)
+
+_The monitoring stack employed for observing the KaaS layer and infrastructure services
+is designed to serve as an **SCS product** in the future versions, once it attains sufficient stability.
+This is because the monitoring focuses on elements like Kubernetes clusters and infrastructure
+services, utilizing Kubernetes API and protocols such as HTTP(S), TCP, ICMP, etc.,
+which are aspects CSPs are unlikely to alter._
```

Comment:
I'm a little lost here what the intention of this sentence is. What do I learn here as a reader? Maybe it's just me/to late at friday afternoon :slightly_smiling_face:

Reply:
Yep, it seems odd to me too after a second read. The section has been removed: 79b0c3d76115c2278b04fe880530c25cb6a4d7fd